### DISCUSSION TITLE
Workflow with multiple initiators

### DISCUSSION THREAD

**Discussion by johnwc** [elsa-workflows/elsa-core#3869] [Q&A] [answered]
Can a workflow have multiple initiator activities? Example: A workflow that has a http endpoint activity as well as a cron activity. So that the workflow can be started manually via a http get or started upon next cron interval.

**Comment by sfmskywalker** [comment, marked as answer]
Yes, you can have multiple triggers. I have one workflow that can be triggered via HTTP requests and Azure Service Bus messages.

**Reply by johnwc** [reply]
How is it setup? Do you have two connections from start going to two different activities, then those two activities connect to one activity that is essentially the first step in the workflow?

**Reply by sfmskywalker** [reply]
Exactly. For example:

<img width="898" alt="image" src="https://user-images.githubusercontent.com/938393/229724962-f1a2e2e8-a0dc-46f0-9b8c-c223775e0fa5.png">
